nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ftp/StandardFTPClientProvider.java

```diff
@@ -152,10 +153,10 @@ private void setClientProperties(final FTPClient client, final PropertyContext c
         client.setRemoteVerificationEnabled(false);
 
         final boolean unicodeEnabled = context.getProperty(UTF8_ENCODING).isSet() ? context.getProperty(UTF8_ENCODING).asBoolean() : false;
-        if (unicodeEnabled) {
-            client.setControlEncoding(StandardCharsets.UTF_8.name());
-            client.setAutodetectUTF8(true);
-        }
+        final Charset charset = unicodeEnabled ? StandardCharsets.UTF_8 : Charset.defaultCharset();
+        client.setCharset(charset);
```

Comment:
Does this change the current behavior, or does the internal character set of the client use `Charset.defaultCharset()` when not otherwise specified? This looks like the right way to go, just wanted to clarify.

Reply:
After evaluating the Apache FTPClient class and parent classes, it appears that the `Charset` property is not used, so it seems best to avoid setting the value.

Reply:
It was a speculative change.  I have no evidence that it makes a difference.  Removed this.